examples/ansys-aedt-workflow/assets/pulse_data.wbjn

```diff
@@ -0,0 +1 @@
+csv_file_name=r'd:/2024_Projects/PyAnsys/pyworkbench-examples/examples/ansys-aedt-workflow/assets/10_1000_Pulse.csv' 
```

Comment:
Never use the absolute path: how would you expect other people to run this example if it refers to absolute path? @pvinod13